core/Scene.lua

```diff
@@ -40,10 +66,10 @@ Scene.new = function (init)
   self.camera = GS:add(OrthographicCamera.new(self.scenegraph, GS[self.scenegraph].root)) --Specify a camera for the scene
   
   --Subscribe to standard events
-  self.message_bus:subscribe(self.gid, "mouse", function() print "Mouse event" end)
-  self.message_bus:subscribe(self.gid, "keyboard", function() print "Key event" end)
-  self.message_bus:subscribe(self.gid, "time", function() print "Time passed" end)
-  self.message_bus:subscribe(self.gid, self.gid, function() print "Got direct event" end)
+  self.message_bus:subscribe(self.gid, "mouse", function() end)
+  self.message_bus:subscribe(self.gid, "keyboard", function() end)
+  self.message_bus:subscribe(self.gid, "time", function() end)
+  self.message_bus:subscribe(self.gid, self.gid, function() end)
```

Comment:
I didn't end up actually using these events in the LudumDare: the elements in the game scene subscribed to this message_bus, but the scene itself didn't have much to do with the event code. Perhaps remove this.